tests/pallas/pallas_test.py

```diff
@@ -253,6 +253,114 @@ def test_pallas_call_out_shape_is_list(self):
     # TODO(necula): we normalize out_shape to a tuple, we shouldn't.
     self.assertIsInstance(res, tuple)
 
+  @jtu.skip_on_devices("gpu")  # TODO: RET_CHECK failure
```

Comment:
Is this TODO resolved?

Reply:
No, it is not resolved, but I see that as separate work.